### PR TITLE
tikv: add storage flow control doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
tikv: add storage flow control doc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6826
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -312,6 +312,35 @@ Configuration items related to the sharing of block cache among multiple RocksDB
 + Default value: 45% of the size of total system memory
 + Unit: KB|MB|GB
 
+## storage.flow-control
+
+Configuration items related the flow control mechanism in TiKV. This mechanism replaces the write stall mechanism in RocksDB and controls flow at the scheduler layer, which avoids the issue of QPS drop caused by the stuck Raftstore or Apply thread when the write traffic is high.
```
Comment (TomShawn) [inline tikv-configuration-file.md:317]:
```suggestion
Configuration items related to the flow control mechanism in TiKV. This mechanism replaces the write stall mechanism in RocksDB and controls flow at the scheduler layer, which avoids the issue of QPS drop caused by the stuck Raftstore or Apply threads when the write traffic is high.
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -312,6 +312,35 @@ Configuration items related to the sharing of block cache among multiple RocksDB
 + Default value: 45% of the size of total system memory
 + Unit: KB|MB|GB
 
+## storage.flow-control
+
+Configuration items related to the flow control mechanism in TiKV. This mechanism supersedes the write stall mechanism in RocksDB and controls flow at the scheduler layer, which avoids the issue of QPS drop caused by the stuck Raftstore or Apply threads when the write traffic is high.
```
Comment (Connor1996) [inline tikv-configuration-file.md:317]:
```suggestion
Configuration items related to the flow control mechanism in TiKV. This mechanism supersedes the write stall mechanism in RocksDB and controls flow at the scheduler layer, which avoids the issue of QPS drop caused by the stuck of Raftstore or Apply threads when the write traffic is high.
```
missing `of`?

**Review by Connor1996 — COMMENTED** [review]
most LGTM

File: tikv-configuration-file.md (hunk 2 above)
Comment (TomShawn) [reply tikv-configuration-file.md:317]: No. "stuck" is an adjective that modifies Raftstore or Apply.

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -312,6 +312,35 @@ Configuration items related to the sharing of block cache among multiple RocksDB
...
++ Default value: `5`
+
+### `l0-files-threshold`
+
++ When the number of kvDB L0 files reaches this threshold, the flow control mechanism starts to work.
++ Default value: `9`
+
+### `soft-pending-compaction-bytes-limit`
+
++ When the pending compaction bytes in KvDB reaches this threshold, the flow control mechanism starts to reject some write requests and reports the `ServerIsBusy` error.
```
Comment (TomShawn) [inline tikv-configuration-file.md:336]:
```suggestion
+ When the pending compaction bytes in KvDB reach this threshold, the flow control mechanism starts to reject some write requests and reports the `ServerIsBusy` error.
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -312,6 +312,35 @@ Configuration items related to the sharing of block cache among multiple RocksDB
...
++ Default value: `"192GB"`
+
+### `hard-pending-compaction-bytes-limit`
+
++ When the pending compaction bytes in KvDB reaches this threshold, the flow control mechanism rejects all write requests and reports the `ServerIsBusy` error.
```
Comment (TomShawn) [inline tikv-configuration-file.md:341]:
```suggestion
+ When the pending compaction bytes in KvDB reache this threshold, the flow control mechanism rejects all write requests and reports the `ServerIsBusy` error.
```

**Review by Connor1996 — APPROVED** [review]

**Comment by hfxsd** [comment]
LGTM

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**